### ISSUE TITLE
Bad link

### ISSUE THREAD

**Issue by powershellshock** [MicrosoftDocs/CloudAppSecurityDocs#150]
I followed the link https://docs.microsoft.com/en-us/cloud-app-security/troubleshoot-docker to get here and the ToC also says this doc is for Docker deployment troubleshooting, but it covers no troubleshooting.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 8ff02fa9-f304-d7c7-18ec-30ebc1c1c03f
* Version Independent ID: 246c3153-cd1b-d7d2-682a-4ff368fdc26c
* Content: [Modify FTP settings](https://docs.microsoft.com/en-us/cloud-app-security/troubleshoot-docker#feedback)
* Content Source: [CloudAppSecurityDocs/troubleshoot-docker.md](https://github.com/Microsoft/CloudAppSecurityDocs/blob/master/CloudAppSecurityDocs/troubleshoot-docker.md)
* Service: **cloud-app-security**
* GitHub Login: @rkarlin
* Microsoft Alias: **rkarlin**

**Comment by rkarlin** [comment]
@ShlomoSagir-MS

**Comment by ShlomoSagir-MS** [comment]
@powershellshock Thanks for your feedback. After investigating, we've discovered that you are correct. Therefore, we have updated the relevant topic to fix this. Thanks again; we always appreciate feedback on our documentation!

For information about troubleshooting docker, see  [Log collector errors](https://docs.microsoft.com/en-us/cloud-app-security/troubleshooting-cloud-discovery#log-collector-errors)
